build.gradle

```diff
@@ -6,8 +6,8 @@ buildscript {
         maven { url 'https://oss.jfrog.org/artifactory/oss-snapshot-local' }
     }
     dependencies {
-        //Do not update the build version
-        //If this version doens't work, go to gradle/wrapper/gradle-wrapper.properties and change distributionUrl gradle version to 6.1.1
+        // Do not update the build version (keep: classpath 'com.android.tools.build:gradle:4.0.2')
```

Comment:
Does this enforce that this dependency must be present?

Also, if let's say we add 
`classpath 'com.android.tools.build:gradle:6.1.1'`
and we don't remove 4.0.2, does Gradle complain?